src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+import mypackage.Color;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int indexColor = new Random().nextInt(Color.values().length);
```

Comment:
Please extract Random to class variable, with current approach you create new object each time method is called what is unnecessary